linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_name_format/rule.yml

```diff
@@ -20,6 +20,7 @@ identifiers:
 
 references:
     ospp: FAU_GEN.1
+    srg: SRG-OS-000039-GPOS-00017
 
 ocil_clause: name_format isn't set to hostname
```

Comment:
``SRG-OS-000039-GPOS-00017`` makes sense. 

Why is the value tied to ``hostname``? Why not ``fqdn`` or ``numeric`` ?

Reply:
Because `hostname` is returned by the gethostname syscall, also this comment is outside the scope of this PR.

Reply:
Hostname is the most user friendly way to name the system. I suppose they could tailor their rules to use FQDN if that is their SOP. The main point is something has to get stashed in the event because by default the machine identifier is not included in the event. So, you need something to tell events apart in the central logging system.

Reply:
Opened https://github.com/ComplianceAsCode/content/issues/5011 to track accepting more than hostname.